website/docs/usage/data-api.md

```diff
@@ -117,12 +116,13 @@ This will load available endpoints into Postman, as shown below:
 
 ![Import Data API Specification into Postman](/img/docs/import-data-api.jpg)
 
-Be sure to set the `{{baseURL}}` to `http://localhost:8080` or the listen address you configured for the Data API.
-
 Using the same examples from earlier, you can now test the endpoints directly in Postman.
-You can also set up environment variables in Postman for the `<username>` and `<password>` to avoid hardcoding them in the requests.
 
 Below are examples of the `insertOne` and `find` endpoints in Postman after importing the OpenAPI spec.
 
 ![Insert a single document](/img/docs/insert-one.jpg)
 ![Find a single document](/img/docs/find-one.jpg)
+
+Be sure to set the `{{baseURL}}` variable to the URL you configured for the Data API (`http://localhost:8080` in this case).
+You can also set up environment variables in Postman for the `<username>` and `<password>` to avoid hardcoding them in the requests.
+See the [Postman documentation](https://learning.postman.com/docs/sending-requests/variables/environment-variables/) for more information on how to set up environment variables.
```

Comment:
I think this information should be above `![Insert a single document](/img/docs/insert-one.jpg)`. Because without these steps, successful insertion and retrieval are not possible. What do you think?